lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -155,9 +155,9 @@ abstract class _BaseAdapter {
   /// Listeners that are registered through native javascript api.
   static final Map<String, html.EventListener> _nativeListeners =
     <String, html.EventListener>{};
-  final html.Element glassPaneElement;
+  final html.Element? glassPaneElement;
   _PointerDataCallback _callback;
-  PointerDataConverter _pointerDataConverter;
+  PointerDataConverter? _pointerDataConverter;
```

Comment:
I couldn't find a situation where this can be null. It's always initialized in the constructor.

Reply:
Done.